epi/src/lib.rs

```diff
@@ -377,34 +377,50 @@ pub const APP_KEY: &str = "app";
 ///
 /// You must enable the "http" feature for this.
 pub mod http {
-    /// A simple http requests.
+    use std::collections::BTreeMap;
+
+    /// A simple http request.
     pub struct Request {
         /// "GET", …
         pub method: String,
         /// https://…
         pub url: String,
-        /// x-www-form-urlencoded body
-        pub body: String,
+        /// The raw bytes.
+        pub body: Vec<u8>,
+        // ("Accept", "*/*"), …
```

Comment:
```suggestion
        /// ("Accept", "*/*"), …
```

Reply:
(made the change outside of this conversation, sorry)